grails-app/controllers/org/pih/warehouse/api/LocationApiController.groovy

```diff
@@ -192,6 +192,11 @@ class LocationApiController extends BaseDomainApiController {
     }
 
     Location bindLocationData(Location location, JSONObject jsonObject) {
+        // We only want to set the supportedActivities if we provide it in the payload
+        if (jsonObject.containsKey("supportedActivities")) {
+            jsonObject.supportedActivities = jsonObject.supportedActivities ?: [ActivityCode.NONE.id]
+        }
```

Comment:
@jmiranda I've used this instead of adding additional `step` param to handle the edge case I've mentioned earlier about going "from the previous step".

Reply:
```
Location.groovy

Boolean isSupportedActivitiesEmpty() { 
   return supportedActivities.empty || (supportedActivities.contains(ActivityCode.NONE) && supportedActivities.size() == 1)) 
```